pyproject.toml

```diff
@@ -47,4 +47,11 @@ build-backend = "poetry.core.masonry.api"
 requires = ["poetry-core"]
 
 [tool.ruff]
-exclude = ["maps", "GameServers"]
+include = ["pyproject.toml", "AIAgent/**/*.py"]
+
+[tool.ruff.lint]
+select = ["F", "E", "W", "B"]
+ignore = ["E501"]
+extend-select = ["E226"]
+preview = true
+explicit-preview-rules = true
```

Comment:
- What are `select` features stand for? Can't find docs on them
- Why do we need experimental features (`preview`) enabled?

not important, just curious: why use `["F", "E", "W", "B"]` specifically? Default `["E4", "E7", "E9", "F"]` is worse?

Reply:
1. `explicit-preview-rules = true` is used to enable specific preview rules, not all of them.

2. The default profile does not cover a wide range of code style violations, so I extended the default set to include more syntax checks.

Reply:
ok